From 813cb4b6b21ef74210224a1034761803938303b1 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Tue, 6 May 2014 11:58:44 +0300
Subject: [PATCH] Delay after insmod

Enable delay after insmod. This will provide the driver time to init and
register wlan0 to be discovered afterwards.

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 wifi/wifi.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wifi/wifi.c b/wifi/wifi.c
index ff9357b..c363c14 100644
--- a/wifi/wifi.c
+++ b/wifi/wifi.c
@@ -223,7 +223,7 @@ int wifi_load_driver()
         return -1;
 
     if (strcmp(FIRMWARE_LOADER,"") == 0) {
-        /* usleep(WIFI_DRIVER_LOADER_DELAY); */
+        usleep(WIFI_DRIVER_LOADER_DELAY);
         property_set(DRIVER_PROP_NAME, "ok");
     }
     else {
-- 
1.7.9.5

